ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -234,33 +244,38 @@
     (% class="col-xs-12 col-sm-4" %)
     (((
       ## leader
-      ; &lt;label for="Meeting.Code.MeetingClass_0_leader"&gt;
-          $!services.icon.render('user') $services.localization.render('contrib.meeting.field.leader')
-        &lt;/label&gt;
-      : $doc.display('leader')
+      #if ($xcontext.action == 'edit' || "$!doc.getValue('leader')" != '')
```

Comment:
There is a lot of duplication. You could also write a macro with some parameters to handle multiple fields.